packages/react/src/vm-extension/node-vm-ext-list.tsx

```diff
@@ -21,38 +21,58 @@ export const NodeVMExtList = (props: NodeVmExtensionListProps) => {
         return inject(BatchDependencyName.NodeService);
     }, []);
 
+    const accountService: AccountService = React.useMemo(() => {
+        return inject(BatchDependencyName.AccountService);
+    }, []);
+
+    const fetchExtensions = React.useCallback(async () => {
+        const accountData = await accountService.get(accountResourceId);
```

Comment:
Should we just add a `getRequired()` function to AccountService which throws if no account is found?

Reply:
This is only for the FakeAccountService, LiveAccountService will throw 404 error when the account is not found.

Reply:
We should try to be consistent across fake and real implementations here. The `AccountService` interface says it can return `undefined`, so the fact that that's not actually the case for `LiveAccountService` is surprising. I like allowing the `get()` to return `undefined`, but that means we should handle the 404 in `LiveAccountService`.

Reply:
Sure, I can make `FakeAccountService` to throw if undefined is to return, then we don't need to check the `undefined` everywhere we use `AccountService`

Reply:
Can we reverse that, and return `undefined` when we see a 404? The nice thing about doing that versus throwing an exception is we'll get help from the compiler (and also a null-check is easier than a try/catch). If we really want a version that throws an exception, that's where I'd implement a `getRequired()` method, but I don't think we really need that.